From 74cd3ea9fb89ad044feba8a3e93f6616c5df7bde Mon Sep 17 00:00:00 2001
From: icg_intg <icg_intg@intel.com>
Date: Thu, 11 Apr 2019 12:23:19 +0800
Subject: [PATCH 1478/1548] isys & psys package lib2600b0 for commit-id 5efdf9c

FW_Manifest_ID: (f9059aea766d4e12819db9a87d3d6f0cdabac0de)

FW_Build_Branch: IPU4_master
FW_Build_Topic: auto_merge_queue
library version: commit-id 5efdf9c
FW Bundle patch number of this release :  397010
FW tag : scci_IPU4_master_20190320_1405

Change-Id: Ie946c0e8af0d3a75f120dbcd52f08b3863c7e6b9
Tracked-On: PKT-2588
Signed-off-by: icg_intg <icg_intg@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h   | 2 +-
 .../lib/DSS_V2_program_group/ia_css_fw_pkg_release.h            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
index 408726c81714..b939cf739802 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20181222
+#define IA_CSS_FW_PKG_RELEASE  0x20190320
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
index 408726c81714..b939cf739802 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/lib2600psys/lib/DSS_V2_program_group/ia_css_fw_pkg_release.h
@@ -11,4 +11,4 @@
 * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
 * more details.
 */
-#define IA_CSS_FW_PKG_RELEASE  0x20181222
+#define IA_CSS_FW_PKG_RELEASE  0x20190320
-- 
2.20.1

